src/main/java/uk/gov/hmcts/reform/sscs/personalisation/Personalisation.java

```diff
@@ -125,6 +126,10 @@
         return personalisation;
     }
 
+    private String getHmctsPhoneNumber(RegionalProcessingCenter rpc) {
+        return rpc != null && rpc.getPhoneNumber() != null ? rpc.getPhoneNumber() : config.getHmctsPhoneNumber();
```

Comment:
I prefer to use `rpc != null && StringUtils.isNotEmpty(rpc.getPhoneNumber())`, that way we would not get blank phone numbers which would break the gov notify template.

Reply:
Have refactored the code so this is not an issue